tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj

```diff
@@ -2,8 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks Condition="'$(OS)' != 'Unix'">net472;net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">net5.0</TargetFrameworks>
+    <TargetFrameworks>net472;net5.0</TargetFrameworks>
```

Comment:
Probably should go back to :
````
    <TargetFrameworks Condition="'$(OS)' != 'Unix'">net472;net5.0</TargetFrameworks>
    <TargetFrameworks Condition="'$(OS)' == 'Unix'">net5.0</TargetFrameworks>
````

Reply:
why is this? non-windows boxes can built fr net472 as long as the reference assembly nuget packages are referenced, shouldn't wr move that direction?

Reply:
Yes something in Visual Studio is making this change automatically, I've seen it before in other branches.  I'll revert but it's a bit strange